### DISCUSSION TITLE
Security Onion Sensor Setup

### DISCUSSION THREAD

**Discussion by ryguy-vt** [Security-Onion-Solutions/securityonion#1756] [Unsupported Versions]
Trying to install a security onion sensor, but the setup say "Install has a problem. Please see /root/sosetup.log for details."

Any ideas? We already set up the master node.

When I grep'd error from the sosetup.log I found "Attempt to authenticate with the salt master failed"

New error's messages:

Errors detected during setup; skipping post-setup steps to allow for analysis of failures
Error response frpm daemon: No such container: so-mysql
Error: No such container: so-mysql

**Comment by jhildreth** [comment]
Were you prompted for the soremote password during setup of the forward node?

I ran into a similar issue, because I had disabled password authentication for ssh on my manager node.
When I (temporarily) allowed password authentication on the manager node, setup of the other nodes prompted for the soremote password (to connect to the manager), and it worked properly.

**Reply by ryguy-vt** [reply]
We first setup the master node, then attempted to setup sensor node but it error's out every time. When setting up the sensor node, it prompts the ssh password for the master node. The setup error's out every time.

We haven't setup any forward nodes yet.

**Reply by ryguy-vt** [reply]
The error messages we see:

Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7@arch=x86_64&repo=os&infra=stock error was 14: curl#6 - "Could not resolve host: mirrorlist.centos.org; Unknown error

error: https://repo.saltstack.com/py3/redhat/7/x86_64/archive/3001.1/SALTSTACK-GPG-KEY.pub: import read failed(2).

Attempt to authenticate with the salt master failed with timeout error

**Reply by jhildreth** [reply]
Gotcha - different issue than what I ran into I guess, unfortunately.

**Reply by ryguy-vt** [reply]
No problem, thank you jhildreth

**Reply by RollingMoses** [reply]
@ryguy-vt , did you find a solution? I'm running into the same issue on my forward node sensor install.

**Reply by sinnetll19** [reply]
Did anybody find a solution to this problem? i have the same issue on forward node sensor install

**Comment by TOoSmOotH** [comment]
Looks like the sensor can't get to the saltstack repo.

**Comment by ryguy-vt** [comment]
We fixed by fine tuning the configuration during install.

**Reply by sinnetll19** [reply]
@ryguy-vt, can you provide more details as to the final fix. i thought my configuration was very specific and it still gave me the same errors as you had originally. i was trying to have a sensor connect to a Manager/storage.

**Reply by stondino00** [reply]
Having same exact issue.  What did you fine tune to fix the so-mysql error?

> 
> We fixed by fine tuning the configuration during install.
